src/components/moderation/moderation.tsx

```diff
@@ -170,19 +170,14 @@ export class Moderation extends Component<MergedProps, ModerationState> {
                 />
                 <div className={styles.submitWrapper}>
                   <div className={styles.characterCounter} data-testid="characterCounter">{`${reportContent.length}/${reportLength}`}</div>
-                  <Tooltip label={tooltipMessage} classNames={styles.tooltip}>
-                    <A11yWrapper onClick={this._handleSubmit}>
-                      <button
-                        role="button"
-                        aria-disabled={submitButtonDisabled}
-                        aria-label={this.props.sendReportLabel}
-                        className={[styles.submitButton, submitButtonDisabled ? styles.disabled : ''].join(' ')}
-                        tabIndex={0}
-                        data-testid="submitButton">
-                        {this.props.sendReportLabel}
-                      </button>
-                    </A11yWrapper>
-                  </Tooltip>
+                  <Button
```

Comment:
what about accessibility, i saw you removed the A11yWrapper component

Reply:
@JonathanTGold see the common PR- he used the a11yWrapper inside the button component- https://github.com/kaltura/playkit-js-common/pull/7/files#diff-b0750a8f9c8bf9268a19aeb46913c3cc020f5bb227fbf42a6e32bf8a57c8b319R53